_component_design/toast.md

```diff
@@ -48,15 +51,20 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/toasts/
 
 <div class="hxRow" markdown="1">
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}
+{% column left:"hxCol hxSpan-8-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
 
-### Best Practices
+Consider the following best practices when you construct toast:
 
-- If there's an error and the user can take an action to correct it, use a call to action (CTA).
-- Use complete sentences. If only one sentence is used, do not include a period. However, if there are two or more sentences, use periods.
-- Try not to exceed more than four lines of your message (excluding the CTA).
-- Only use one CTA per toast. Never include “Dismiss” or “Close” as a CTA.
-- Clicking on the “X” closes the toast notification. Hovering over it keeps it open until the cursor moves off of it. (See [Timing]({{page.url}}#timing) section for more detail.)
+- Construct the toast so that the user can navigate away and still be notified in their current context when the user-initiated process completes or fails.
+- A toast can include an optional call-to-action (CTA) that navigates the user out of the current page, or that triggers an action.
+- Only use one CTA for a toast. Do not include a **Dismiss** or **Close** CTA.
+- If there is an error and the user can take an action to correct, include a CTA.
+- Use human-friendly, complete sentences when you write the notification message. For more information about writing notification message, see [Alert and Message Guidelines]({{site.baseurl}}/style/alert-messages.html).
+- The notification message should not exceed more than four lines, excluding the CTA.
+- The user can click close **X** to close the toast.
+- A toast remains open when the user hovers over it and closes when the cursor moves off of it. Refer to [Behaviors]({{page.url}}#behaviors) for more information about toast timing.
```

Comment:
off of it, provided the full 10 seconds has elapsed.

Reply:
Done.